packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -254,7 +268,9 @@ export function PrivateBlockToolbar( {
 					</>
 				) }
 				<BlockEditVisuallyButton clientIds={ blockClientIds } />
-				<BlockSettingsMenu clientIds={ blockClientIds } />
+				{ ! isZoomOutMode && (
+					<BlockSettingsMenu clientIds={ blockClientIds } />
+				) }
```

Comment:
Personally I wonder if we should instead keep the blockSettingsMenu and try to figure why the default "trash" button is not visible there instead of forking the experience in zoom-out.

Reply:
It's probably that all the other buttons should be removed, so that Delete is the only thing left, is which case the more toggle shouldn't be there?

Reply:
Why would we want to remove "duplicate" for instance? 

Also, if we want to remove the dropdown if there's only one button, should this behavior be built-in into the toolbar or specific to zoom-out?